site/README.md

```diff
@@ -0,0 +1,138 @@
+# Docsy Example
```

Comment:
Update this file.

Or maybe you are waiting for us to get a domain first?

Reply:
Waiting to get a domain first.